### ISSUE TITLE
Big Loop Can't be Split into two Small Loops

### ISSUE THREAD

**Issue by DNAcer** [NATuG3/NATuG3#5]
If you have one strand forming a big loop (such as the red loop in the first image below), and you try to make a junction between two points on that strand (such as by clicking on the middle NEMiDs in the loop), it should split into two smaller loops. Instead it gets tangled up. See second image.
<img width="342" alt="Screen Shot 2022-11-19 at 1 08 01 AM" src="https://user-images.githubusercontent.com/118033291/202837238-7427487a-bd56-4a26-a708-d79cf1b48cba.png">
<img width="342" alt="Screen Shot 2022-11-19 at 1 08 13 AM" src="https://user-images.githubusercontent.com/118033291/202837240-d1a68bfb-bb90-4d9e-8c12-e252f3b4accf.png">

**Comment by DNAcer** [comment]
@404Wolf  This issue has been marked resolved, but it still remains.

**Comment by 404Wolf** [comment]
@DNAcer 
I've just pushed a fix for this to main, and it seems to be working well on my end. However, if you could double check that it works for you before I close this that'd be great.

**Comment by DNAcer** [comment]
Hi Wolf,

I haven't had any problems with splitting up closed loops and reclosing and
resplitting. Great work!

However, I have found that any time I manually set the color of a strand,
the NEMid triangles generally get larger, but the junctable NEMid triangles
get smaller. The problem goes away if I set the color to autoselect again,
but it doesn't seem to go away otherwise, as far as I have been able to
tell.

Cheers,
Bill
